code/headers/controller_interface_c.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+#include <joystick_value_s.hpp>
+
+namespace R2D2::manual_control {
+    
+    enum buttons {button_a, button_b, button_x, button_y, dpad_up, dpad_down, dpad_left, dpad_right, bumper_left, bumper_right, menu_left, menu_right};
+    enum sliders {slider_l, slider_r};
+    enum joysticks {joystick_l, joystick_r};
+
+
+    /**
+ *  \brief
```

Comment:
The indents here look weird, it does this in the whole document, please check this for the whole project